prescience_client/commands/start_command.py

```diff
@@ -458,3 +460,156 @@ def exec(self, args: dict):
             mask_id=mask_id,
             selected_column=columns
         )
+
+class StartAutoML(Command):
+    def __init__(self, prescience_client):
+        super().__init__(
+            name='auto-ml',
+            help_message='Start an Auto-ML task, which will chain all needed tasks until model deployment',
+            prescience_client=prescience_client,
+            sub_commands=[]
+        )
+
+    def init_from_subparser(self, subparsers, selector):
+        super().init_from_subparser(subparsers, selector)
+        self.cmd_parser.add_argument('source-id', type=str, nargs='?', help='The source from which you want to start a AutoML task')
+
+        self.cmd_parser.add_argument('--dataset-id', type=str, help='Name you want for the created dataset')
+        self.cmd_parser.add_argument('--model-id', type=str, help='Name you want for the created model')
+
+
+        self.cmd_parser.add_argument('--problem-type', type=ProblemType, choices=list(ProblemType),
+                                       help=f"Type of problem for the dataset (default: {DEFAULT_PROBLEM_TYPE})",
+                                       default=DEFAULT_PROBLEM_TYPE)
+        self.cmd_parser.add_argument('--time-column', type=str,
+                                       help='Identifier of the time column for time series. Only for forecasts problems.')
+        self.cmd_parser.add_argument('--nb-fold', type=int, help='How many folds the dataset will be splited')
+        self.cmd_parser.add_argument('scoring-metric', nargs='?', type=ScoringMetric, choices=list(ScoringMetric),
+                                     help=f'The scoring metric to optimize on. If unset it will trigger the interactive mode.')
+        self.cmd_parser.add_argument('--budget', type=int,
+                                     help='Budget to allow on optimization (default: it will use the one configure on prescience server side)')
+        self.cmd_parser.add_argument('--watch', default=False, action='store_true',
+                                     help='Wait until the task ends and watch the progression')
+        self.cmd_parser.add_argument('--forecast-horizon-steps', type=int,
```

Comment:
uniformize forecast name between function.
In optimize use forecasting_horizon-step / in ux it's forecasting_discount ....